api-reference/50 Common/utils/ui/notify(message_stack).md

```diff
@@ -0,0 +1,136 @@
+---
+id: ui.notify(message,stack)
+module: ui/notify
+export: default
+---
+---
+##### shortDescription
+Creates a stackable toast message.
+
+##### param(message): String
+The message's text.
+
+##### param(stack): Object
+An object with the following structure: *{position, direction}*.
+
+---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos//WidgetsGallery/Demo/Toast/Stack/"
+}
+
+To stack toast messages, specify the `position` field in the **stack** object.
+
+The `position` field can be one of the following:
+
+- String    
+Available values:
+
+    - *'top left'*
+
+    - *'top center'*
+
+    - *'top right'*
+
+    - *'left center'*
+
+    - *'center'*
+
+    - *'right center'*
+
+    - *'bottom left'*
+
+    - *'bottom center'*
+
+    - *'bottom right'*
+
+- Object    
+Avaliable values: *{left | right, top | bottom}*.
+
+[note]
+
+If you use an object for `position` field, you can only specify one field in that object for vertical coordinate and one field for horizontal coordinate (for example, if you specify *'top'*, you do not need to specify *'bottom'*).
+
+[/note]
+
+You can also specify the `direction` field in the **stack** object. It is a string that can be one of the following:
+
+- *'up'*
+
+- *'down'*
+
+- *'left'*
+
+- *'right'*
+
+- *'up-reverse'*
```

Comment:
Can we still change these names? Later in the topic, you explain the difference well. You use the words "push" and "stack" that make much more sense to me than "up" and "up-reverse" (that just sounds like a bad name for "down" :-) ). So it would be great if we can change these options to `up-push`, `up-stack`, `down-push`, `down-stack`, and so on.

(This comment applies to the other topic as well.)

Reply:
I decided not to change these names.
The main reason is I think that the 'reverse' postfix is familiar to our customers. It is used in [CSS](https://developer.mozilla.org/ru/docs/Web/CSS/flex-direction) and in [other UI libraries](https://mui.com/material-ui/react-stack/). The other thing, we already use 'stack' as an option name, and I think it's a bit odd to use it 2 times in the same property.

Reply:
I don't like `up` and `up-push` for the reason that it does not convey what `up` means. I'm okay with `up-push`, but I think we need another word for `up-reverse`.
I tried to come up with suggestions, please check them out:
`up-load` (I know it sounds like upload, but still...)
`up-layer`
`up-compile`
`up-extend`

Reply:
@vladaskorohodova 

I was thinking about  `up-add` or 'up-append' as an opposite to `up-push`. 

I still think the best choice would be `up-stack`. "Stack" is what kids do when they put blocks on top of each other. I think that word describes the process best. You didn't want to have "stack" in both the option name and the option value though. 

In your versions, I was thinking about `up-layer`, but I would always think of 'layer' as a noun first. `up-extend` is probably the best choice from your list. 

So I would go with `up-add`, `up-append`, or `up-extend`. My personal favorite is `up-append`, but I can live with any of the three.

Reply:
We had a chat with Anna Kondratova and decided to use `up-push` and `up-stack`. I guess I can live with 'stack' being used 2 times...